docs/guide/usage/mqtt_topics_and_messages.md

```diff
@@ -82,6 +82,42 @@ Publishing messages depends on the MQTT client you use. For example to publish a
  mosquitto_pub -t 'zigbee2mqtt/0x0fffffffffffffff/set' -m '{ "state": "ON" }'
 ```
 
+#### Directly reading/writing ZCL attributes
+For development/debug purposes it's possible to directly send read/write attribute requests to a device through this endpoint.
+
+##### Read requests:
+
+```js
+{
+  "read": {
+    "cluster": genBasic, // Either name (if defined in standard) or ID of cluster can be used
+    "attributes": [1]    // Either name (if defined in standard) or ID of attribute can be used
+  }
+}
+```
+##### Write requests:
+
```

Comment:
I think it would be good to link to the clusters; https://github.com/Koenkk/zigbee-herdsman/blob/master/src/zspec/zcl/definition/cluster.ts

Reply:
added!